ts/tsconfig.json

```diff
@@ -3,12 +3,20 @@
     "compilerOptions": {
         "target": "es6",
         "module": "es6",
-        "lib": ["es2017", "es2019.array", "es2018.promise", "dom", "dom.iterable"],
+        "lib": [
+            "es2017",
+            "es2020.string",
```

Comment:
matchAll() is not supported on iOS 12: https://caniuse.com/mdn-javascript_builtins_string_matchall

Reply:
(globalThis is also iOS 12.2+, but is at least supported by the later patch releases)